packages/cactus-plugin-ledger-connector-corda/src/main/typescript/plugin-ledger-connector-corda.ts

```diff
@@ -54,6 +56,7 @@ export interface IPluginLedgerConnectorCordaOptions
   cordaStartCmd?: string;
   cordaStopCmd?: string;
   apiUrl?: string;
+  privateKeyPath?: string;
```

Comment:
@elenaizaguirre Please write some JSdoc code documentation for the property specifically here explaining what this does, make it clear that if specified, it will override the value specified in `sshConfigAdminShell`. Also mention that it's optional for the reason that it's not recommended using it.

In the meantime I'll try to do some research into how we could avoid the whole problem with some bash sorcery.

Reply:
Done